test/rulesets/Base/array.jl

```diff
@@ -105,3 +105,15 @@ end
     test_rrule(fill, 44.0, 4; check_inferred=false)
     test_rrule(fill, 2.0, (3, 3, 3))
 end
+
+@testset "extrema" begin
+    @testset "$f" for f in [maximum, minimum]
+        test_rrule(f, rand(10))
+        test_rrule(f, rand(3,4))
+        test_rrule(f, rand(3,4), fkwargs=(dims=1,))
+        test_rrule(f, rand(3,4,5), fkwargs=(dims=(1,3),))
+        test_rrule(f, rand(1))  # both extrema are the same index
+        @test_skip test_rrule(f, Float64[1,2,-1,-2,0,2,-2])  # attains max twice -- finite diff picks another subgradient
```

Comment:
Here I think FiniteDifferences gives the symmetric subgradient, like `[0, 0.5, 0, 0, 0, 0.5, 0]`.

Reply:
Now there's a version which takes the symmetric convention. Not sure it's worth it.